Libraries/Components/StatusBar/StatusBar.js

```diff
@@ -164,6 +201,11 @@ const StatusBar = React.createClass({
       StatusBarManager.setNetworkActivityIndicatorVisible(visible);
     },
 
+    /**
+     * Set the background color for the status bar
+     * @param {string} color Background color.
+     * @param {boolean=} animated Animate the style change.
```

Comment:
Actually, for components, the type is being parsed from Flow - it should take precedence over the one declared via jsdoc. The things that's happening here is just making sure the description is added and therefore matching up the parameter name. 

I had checked jsdoc to see if we could skip the type when describing the param but it seems one cannot. This is a valid concern that we should revisit.